src/main/java/me/mrCookieSlime/ExoticGarden/ExoticGarden.java

```diff
@@ -146,9 +146,21 @@ public void onEnable() {
+		registerMagicalPlant("Copper", new CustomItem(SlimefunItems.COPPER_DUST, 8),  "eyJ0aW1lc3RhbXAiOjE1ODI2ODUzNTY5NDMsInByb2ZpbGVJZCI6ImIxZDI3NGViOGU0YjRmNzU4NDJkY2Y3MzU0ZTZlZTFhIiwicHJvZmlsZU5hbWUiOiJUaGF0UHJlc3RvbiIsInRleHR1cmVzIjp7IlNLSU4iOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS9kNGZjNzJmM2Q1ZWU2NjI3OWE0NWFjOWM2M2FjOTg5NjkzMDYyMjdjM2Y0ODYyZTljN2MyYTQ1ODNjMDk3YjhhIn19fQ==",
 		new ItemStack[] {null, SlimefunItems.COPPER_DUST, null, SlimefunItems.COPPER_DUST, getItem("GOLD_PLANT"), SlimefunItems.COPPER_DUST, null, SlimefunItems.COPPER_DUST, null});
 
+		registerMagicalPlant("Aluminum", new CustomItem(SlimefunItems.ALUMINUM_DUST, 8),  "eyJ0aW1lc3RhbXAiOjE1ODI2ODUyNjY1ODYsInByb2ZpbGVJZCI6ImIxZDI3NGViOGU0YjRmNzU4NDJkY2Y3MzU0ZTZlZTFhIiwicHJvZmlsZU5hbWUiOiJUaGF0UHJlc3RvbiIsInRleHR1cmVzIjp7IlNLSU4iOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS9mNDQ1NTM0MWVhZmYzY2Y4ZmU2ZTQ2YmRmZWQ4ZjUwMWI0NjFmYjZmNmQyZmU1MzZiZTdkMmJkOTBkMjA4OGFhIn19fQ==",
+		new ItemStack[] {null, SlimefunItems.ALUMINUM_DUST, null, SlimefunItems.ALUMINUM_DUST, getItem("GOLD_PLANT"), SlimefunItems.ALUMINUM_DUST, null, SlimefunItems.ALUMINUM_DUST, null});
+
+		registerMagicalPlant("Tin", new CustomItem(SlimefunItems.TIN_DUST, 8),  "eyJ0aW1lc3RhbXAiOjE1ODI2ODU0NjI4MDYsInByb2ZpbGVJZCI6ImIxZDI3NGViOGU0YjRmNzU4NDJkY2Y3MzU0ZTZlZTFhIiwicHJvZmlsZU5hbWUiOiJUaGF0UHJlc3RvbiIsInRleHR1cmVzIjp7IlNLSU4iOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS82ZWZiNDNiYTJmZTY5NTkxODBlZTczMDdmM2YwNTQ3MTVhMzRjMGEwNzA3OWFiNzM3MTI1NDdmZmQ3NTNkZWRkIn19fQ==",
+		new ItemStack[] {null, SlimefunItems.TIN_DUST, null, SlimefunItems.TIN_DUST, getItem("GOLD_PLANT"), SlimefunItems.TIN_DUST, null, SlimefunItems.TIN_DUST, null});
+		
+		registerMagicalPlant("Silver", new CustomItem(SlimefunItems.SILVER_DUST, 8),  "eyJ0aW1lc3RhbXAiOjE1ODI2ODUzOTk4NzAsInByb2ZpbGVJZCI6ImIxZDI3NGViOGU0YjRmNzU4NDJkY2Y3MzU0ZTZlZTFhIiwicHJvZmlsZU5hbWUiOiJUaGF0UHJlc3RvbiIsInRleHR1cmVzIjp7IlNLSU4iOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS8xZGQ5NjhiMTg1MWFhNzE2MGQxY2Q5ZGI3NTE2YThlMWJmN2I3NDA1ZTUyNDVjNTMzOGFhODk1ZmU1ODVmMjZjIn19fQ==",
+		new ItemStack[] {null, SlimefunItems.SILVER_DUST, null, SlimefunItems.SILVER_DUST, getItem("GOLD_PLANT"), SlimefunItems.SILVER_DUST, null, SlimefunItems.SILVER_DUST, null});
+		
+		registerMagicalPlant("Lead", new CustomItem(SlimefunItems.LEAD_DUST, 8),  "eyJ0aW1lc3RhbXAiOjE1ODI2ODU0MzIyMjYsInByb2ZpbGVJZCI6ImIxZDI3NGViOGU0YjRmNzU4NDJkY2Y3MzU0ZTZlZTFhIiwicHJvZmlsZU5hbWUiOiJUaGF0UHJlc3RvbiIsInRleHR1cmVzIjp7IlNLSU4iOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS85M2MzYzQxODAzOWM0YjI4YjBkYTc1YTZkOWIyMjcxMmM3MDE1NDMyZDRmNDIyNmQ2Y2MwYTc3ZDU0YjY0MTc4In19fQ==",
+		new ItemStack[] {null, SlimefunItems.LEAD_DUST, null, SlimefunItems.LEAD_DUST, getItem("GOLD_PLANT"), SlimefunItems.LEAD_DUST, null, SlimefunItems.LEAD_DUST, null});
```

Comment:
I don't think that these should require a gold plant.
As Aluminum, Tin, Silver and Lead all use iron ingots as textures, they should maybe require an Iron Plant instead.

Reply:
Ok, I changed it to iron plants